### PR TITLE
Now the view of Materials is updated

### PR DESCRIPTION
**Pull request by KuroScar42**

### CODE REVIEW

File: src/components/views/Materials/Materials.tsx [hunk 1]
```diff
@@ -67,80 +63,123 @@ export default function Materials() {
...
+            style={{ margin: '0 10px 0 0', maxWidth: '500px' }}
+            placeholder={appStrings.search}
+            onChange={handleSearch}
+          ></SearchInput>
+          <div style={{ textAlign: 'end' }}>
+            <Button onClick={() => setIsModalOpen(true)}>+</Button>
+            <Modal isOpen={isModalOpen} onClose={() => setIsModalOpen(false)}>
+              <Heading as="h2" size="lg">
+                {selectedMaterial.id
+                  ? appStrings.editProject
```
Comment (EstebanVU) [inline src/components/views/Materials/Materials.tsx:137]: in this case should be editMaterial and addMaterial

File: src/services/materialsService.ts [hunk 2]
```diff
@@ -44,22 +44,40 @@ export const getMaterials = async (): Promise<IMaterialBreakdown[] | null> => {
...
+export const addMaterial = async ({
+  material,
+  appStrings,
+  successCallback,
+  errorCallback,
+}: {
+  material: IMaterial;
+} & IService) => {
   try {
     const { cost, name, unit } = material;
```
Comment (EstebanVargasG360) [inline src/services/materialsService.ts:56]: Better approach is to use object destructuring. in this case {id, ...rest} = material

Comment (EstebanVU) [inline src/services/materialsService.ts:56]: Better approach is to use object destructuring. in this case {id, ...rest} = material

File: src/services/materialsService.ts [hunk 3]
```diff
@@ -44,22 +44,40 @@ export const getMaterials = async (): Promise<IMaterialBreakdown[] | null> => {
...
+export const updateMaterial = async ({
+  material,
+  appStrings,
+  successCallback,
+  errorCallback,
+}: {
+  material: IMaterial;
+} & IService) => {
   try {
     const { cost, name, unit, id } = material;
```
Comment (EstebanVU) [inline src/services/materialsService.ts:74]: same here

File: src/services/materialsService.ts (hunk 2 above)
Comment (KuroScar42) [reply src/services/materialsService.ts:56]: If I use that way it throw a error with the variables 
image

Comment (EstebanVU) [reply src/services/materialsService.ts:56]: you need to pass rest as parameter of addDoc like addDoc(materialDocRef, rest);

**Pull request merged**